### PR TITLE
Update log status mapping to RFC 5424

### LINKED ISSUE #62: Log level integer should follow RFC 5424
## Current behaviour

The current mapping is https://github.com/ivx/ink/blob/master/lib/ink.ex#L241-L244:

```elixir
  defp level(level) do
    case level do
      :debug -> 20
      :info -> 30
      :warn -> 40
      :error -> 50
    end
  end
```

## Expected behaviour

The log level integer should follow The Syslog Protocol ([RFC 5424](https://tools.ietf.org/html/rfc5424)) as also [used by Erlang](http://erlang.org/documentation/doc-10.0/lib/kernel-6.0/doc/html/logger_chapter.html#log-level) and most if not all other systems.

Level | Integer | Description
-- | -- | --
emergency | 0 | system is unusable
alert | 1 | action must be taken immediately
critical | 2 | critical conditions
error | 3 | error conditions
warning | 4 | warning conditions
notice | 5 | normal but significant conditions
info | 6 | informational messages
debug | 7 | debug-level messages

### PR DESCRIPTION
**Pull request by unnawut**
Closes #62 

Not sure though if the existing mapping was specific to some use case. If so, feel free to let me know, we could turn this into a flag as well.

### CODE REVIEW

**Comment by mmainz** [comment]
I answered you here: https://github.com/ivx/ink/issues/62#issuecomment-646034161

TLDR: I would accept a config option that enables this.

**Comment by unnawut** [comment]
Thank you for the feedback! I'll come back with a new PR when ready.
